pywbem/cim_obj.py

```diff
@@ -5298,13 +5320,23 @@ def __init__(self, name, type, reference_class=None, is_array=None,
             `None` is interpreted as an empty set of qualifiers.
 
           value:
-            Deprecated: Because the object represents a parameter declaration,
-            this parameter does not make any sense. Specifying a value other
-            than `None` will cause a :term:`DeprecationWarning` to be issued.
+            The value of the CIM method parameter for the method invocation.
+            Has no meaning for parameter declarations.
 
             The specified value will be converted to a :term:`CIM data type`
             using the rules documented in the description of
             :func:`~pywbem.cimvalue`, taking into account the `type` parameter.
+
+          embedded_object (:term:`string`):
+            A string value indicating the kind of embedded object represented
+            by the parameter value. Has no meaning for parameter declarations.
+
+            For details about the possible values, see the corresponding
+            attribute.
+
+            `None` means that the value is unspecified, causing the same-named
+            attribute in the ``CIMParameter`` object to be inferred. An
```

Comment:
I would guess that the reader would ask at this point what we mean by 'inferred'

Reply:
DONE - clarified that it is inferred from the parameter value.